### ISSUE TITLE
Separate ENV file

### ISSUE THREAD

**Issue by felix-andreas** [emittance/accelduck#2]
Waere es nicht eine "saubere" Loesung die Enviroment Varibales statt in `.bashrc` in eine `.env` zu packen? Das file wuerde dann bei starten des Containers mit `--env-file`
```docker run ... --env-file accelduck.env ...```
geladen werden.

**Vorteile:**
1. Wenn man ein Programm (z.B. elegant) mit `docker exec accelduck elegant` ausfuehrt, waeren das ENV schon geladen. Und man muss nicht die einzelnen Variablen mit `docker exec -e VAR` uebergeben! (Im falle von elegant den `RPNDEFNS` Pfad )??
2. Man waere nicht von Bash abhaengig und koennte auch eine andere Shell benutzen.

Ich habe bereits so ein file erstellt! Ein Problem dabei ist noch, dass ich nicht weiss, wie man Variablen erweitertet. Also `PATH=$PATH:...` funktioniert nicht!

```
LIBGL_ALWAYS_SOFTWARE=1
XDG_RUNTIME_DIR=/tmp

#### EPICS-BASE ####
EPICS_CA_ADDR_LIST='193.149.12.5 193.149.12.6'
EPICS_HOST_ARCH=linux-x86_64
HOST_ARCH=linux-x86_64
EPICS_BASE=/opt/epics/base-3.15.6/
# PATH=$PATH:/opt/epics/base-3.15.6/bin/linux-x86_64/ # auskommentiert, da sonst PATH ueberschrieben wird

#### Elegant ####
RPN_DEFNS=/opt/elegant/defns.rpn

#### Ocelot ####
# PYTHONPATH=$PYTHONPATH:/opt/ocelot

#### accpy ####
# PYTHONPATH=$PYTHONPATH:/opt/accpy

#### julia ####
# PATH=$PATH:/opt/julia/julia-1.1.0/bin/

#### pycharm ####

#### R ####
R_LIBS='/opt/R/Rpackages/'

#### PROSILICA ####
GENICAM_GENTL64_PATH=/opt/vimba/Vimba_2_1/VimbaGigETL/CTI/x86_64bit
GENICAM_GENTL32_PATH=/opt/vimba/Vimba_2_1/VimbaGigETL/CTI/x86_32bit

```

**Comment by farmborst** [comment]
Yes, I agree. It would be useful having the enviroment varibales available independently. Docker has some confusing infos on the env-file. Here (https://docs.docker.com/compose/env-file/) they state: "Environment variables defined in the .env file are not automatically visible inside containers."

Maybe the entrypoint script could be used. I will look into this. But please feel free to give me a working solution.

**Comment by farmborst** [comment]
Here a workaround for your problem as I understand it:

- your problem:
> `>> docker exec -it 3b20b5c946e7 bash -c "julia --version"`
> bash: julia: command not found
- workaround:
>` >> docker exec -it 3b20b5c946e7 bash -c "source /home/sciduckusr/.bashrc && julia --version"`
>julia version 1.1.1
